### DISCUSSION TITLE
4th option of CQ19 and or max Target size

### DISCUSSION THREAD

**Discussion by quebecman** [RandomEngy/VidCoder#811] [General]
Hello,

 I know this sounds a silly discussion but is there anyway to have a fourth option in VIDCODER displaying CQ19 +and or MAX Target Size of mb.

  Like said before, I have been using HB since a long time and it always drives me insane when I wake up and HB has created a TV show of an SD at 16 gigs for only a thirteen episodes.  Wich when choosing Target size of X mb you get max of 5 gigs TV show.

As per example, always be under 450megs at CQ19.  As per HB documentation, CQ is considered to be the best from HB for quality and you don't have to do a second pass.

The thing is you never know how big the file will be with CQ.  It's a hit or miss.

Thanx. Regards.

**Comment by RandomEngy** [comment]
The point is that it's better to know how good the picture will look than it is to know how big the file will be. You might not know how much data it takes to encode something but that's fine. It might also be that you can't tell the difference between CQ 19 and CQ 20, which could explain why the files are a bit large in some cases.

I don't plan on adding any "hybrid" options.

**Comment by quebecman** [comment]
Hmmm... I don't know if this is only a question of luck but since the last two update of the NIGHTLY version, I have seen a drastic difference in the release size of the CQ19 for the better.
 
 I have redone some episodes that where bloatware and now, they are just fine.  Pasted from 650 megs to 375... wich is big when you have a 24 episodes.
